content/applications/finance/payment_providers/mollie.rst

```diff
@@ -8,14 +8,14 @@ Configuration
 =============
 
 .. seealso::
-   - :ref:`payment_providers/add_new`
+   - :ref:`payment-providers-add-new`
 
 Credentials tab
 ---------------
 
-Odoo needs your **API Credentials** to connect with your Mollie account, which comprise:
-
-- **API Key**: The test or live API Key depending on the configuration of the provider.
+Odoo needs your **API Credentials** to connect with your Mollie account, including your
+:guilabel:`**API Key`, i.e., the test or live API Key depending on the configuration of the
```

Comment:
It sounds like the API is only one of the credentials that Odoo needs.

Reply:
I'm a bit confused, am I missing something? I can only see one field in the Credentials tab for Mollie. Having just bullet point felt a bit odd, so I reworded the sentence a little bit (I just came accross it while changing the refs).

Reply:
No, I mean that the instructions make it sound like there exist other API credentials fields: "API credentials [...] including your API key"

Reply:
(I will take care of this in another PR as the whole doc page needs work)